src/scripts/main.ts

```diff
@@ -5,14 +5,19 @@ import {
 } from "../config/index";
 import {
     getOrgData,
-    getTeamsData,
-    setOrgData
+    setOrgData,
+    setTeamsData
 } from "../utils/index";
+import { getTeamsData } from "../service/github";
 
 export const main = async (org: string): Promise<void> => {
     try {
         await getOrgData(org);
-        await getTeamsData();
+
+        // Reduce complexity of getOrgData and setOrgData by refactoring data getting and setting of repos, teams and names into their own functions. Store output of api-sdk response in variables to remove need to use TMP_DATA
+
+        const teams = await getTeamsData(`https://api.github.com/orgs/${org}/teams`);
```

Comment:
I would have created a constant variable, something like `const TEAMS_URL = https://api.github.com/orgs/${org}/teams`

Reply:
Not implemented as user provides `org` variable, discussed on card [NTRNL-235](https://technologyprogramme.atlassian.net/browse/NTRNL-235?focusedCommentId=136891&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-136891)